config.h.in

```diff
@@ -84,6 +84,46 @@
 #cmakedefine HAVE_FLOCK
 #cmakedefine HAVE_STRPTIME
 
+#cmakedefine INDEPENDENT_ENVIRONMENTVARIABLE_PROBE
```

Comment:
OPENSCAP_PROBE_*** would be a better scheme IMO, less clashing, prefix is better than suffix IMO

Reply:
OK, I'll change that.